tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/util/ProjectImportUtil.java

```diff
@@ -549,99 +587,64 @@ public static IStatus validateSDKPath(final String currentPath)
                             return ProjectCore.createErrorStatus("Can't find sdk in workspace");
                         }
 
-                        retVal = sdk.validate();
+                        retVal = sdk.validate( true );
                     }
                     else
                     {
-                        retVal = ProjectCore.createErrorStatus( "SDK does not exist." );
+                        retVal = ProjectCore.createErrorStatus(
+                            "Could not determine SDK from project location " + currentPath );
                     }
                 }
             }
+            else
+            {
+                retVal = ProjectCore.createErrorStatus( "Invalid project location" );
+            }
         }
 
         return retVal;
     }
 
-    public static IStatus validateSDKProjectPath(final String currentPath)
+    public static IStatus validateWorkspacePath(final String currentPath)
     {
-        IStatus retVal = Status.OK_STATUS;
+        IStatus retVal = validatePath( currentPath );
 
-        if( !org.eclipse.core.runtime.Path.EMPTY.isValidPath( currentPath ) )
+        if( retVal.isOK() )
         {
-            retVal = ProjectCore.createErrorStatus( "\"" + currentPath + "\" is not a valid path." );
-        }
-        else
-        {
-            IPath osPath = org.eclipse.core.runtime.Path.fromOSString( currentPath );
+            LiferayWorkspaceUtil.clearWorkspace( currentPath );
```

Comment:
This shouldn't be called in a method call "validateWorkspacePath".  a call to "validate" should be deleting files, etc.